kalite/distributed/middleware.py

```diff
@@ -16,3 +18,23 @@ def process_request(self, request):
             and not request.path.startswith(settings.STATIC_URL)):
 
             raise PermissionDenied()
+
+
+class LogRequests:
+
+    def process_response(self, request, response):
+        logger = logging.getLogger('django.request')
+        # This is added because somehow requests aren't being logged
+
+        IGNORED_PATHS = (
```

Comment:
Possibility to ignore requests that are known to be made extremely often so we don't max out the log file. We should use log rotation, though.